docs/EN_US/ContainerizedHPCC/ContainerizedMods/TroubleshootingHelmDeployments.xml

```diff
@@ -0,0 +1,516 @@
+
+    <para>Use the <emphasis role="strong">kubectl</emphasis> command-line tool
+    to get detailed information about pods. By describing a pod, you can gain
+    valuable insights into its current state, configuration, and resource
+    utilization.</para>
+
+    <para>To get detailed information about a pod, use the following command
+    in a terminal window:</para>
+
+    <para><programlisting>kubectl describe pod &lt;pod-name&gt;</programlisting></para>
+
+    <para>The output provides detailed information about the pod,
+    including:</para>
+
+    <para><variablelist>
+        <varlistentry>
+          <term>Events</term>
+
+          <listitem>
+            <para>A timeline of events related to the pod's lifecycle. If
+            there are issues with the deployment, they are commonly found in
+            this section.</para>
+          </listitem>
+        </varlistentry>
+
+        <varlistentry>
+          <term>Containers</term>
+
+          <listitem>
+            <para>Information about the containers running within the
+            pod.</para>
+          </listitem>
+        </varlistentry>
+
+        <varlistentry>
+          <term>Status</term>
+
+          <listitem>
+            <para>The current status of the pod.</para>
+          </listitem>
+        </varlistentry>
+
+        <varlistentry>
+          <term>Conditions</term>
+
+          <listitem>
+            <para>The conditions that the pod must meet to be considered
+            running.</para>
+          </listitem>
+        </varlistentry>
+      </variablelist></para>
+
+    <para>If you have deployments to more than one namespace, use this
+    command:</para>
+
+    <para><programlisting>kubectl describe pod &lt;pod-name&gt;  -A</programlisting></para>
+
+    <para>This describes the pod across all namespaces.</para>
+
+    <para>By carefully analyzing this information, you can:</para>
+
+    <para><variablelist>
+        <varlistentry>
+          <term>Identify and troubleshoot issues</term>
+
+          <listitem>
+            <para>Pinpoint the root cause of problems, such as resource
+            constraints, configuration errors, or network connectivity
+            issues.</para>
+          </listitem>
+        </varlistentry>
+
+        <varlistentry>
+          <term>Monitor pod health and performance</term>
+
+          <listitem>
+            <para>Track the pod's status, resource usage, and event history to
+            ensure it's operating as expected.</para>
+          </listitem>
+        </varlistentry>
+
+        <varlistentry>
+          <term>Optimize resource allocation</term>
+
+          <listitem>
+            <para>Adjust resource requests and limits to improve performance
+            and cost-efficiency.</para>
+          </listitem>
+        </varlistentry>
+
+        <varlistentry>
+          <term>Debug application behavior</term>
+
+          <listitem>
+            <para>Examine container logs and events to understand the
```

Comment:
I think this is under "By carefully analyzing this information, you can:" , suggesting you can see logs via kubectl describe pod.. - a tad misleading.. since kubectl describe will not show you any logging insights.
You need to run kubectl logs <podname> for that...

Maybe delete this listitem, since you have a later section covering logging anyway.